account_financial_report_qweb/report/abstract_report_xlsx.py

```diff
@@ -360,7 +360,7 @@ def _get_currency_amt_header_format(self, line_object):
     def _generate_report_content(self, workbook, report):
         pass
 
-    def _get_report_name(self):
+    def _get_report_name(self, objects):
```

Comment:
wouldn't be better to put a better name than 'objects' ? Why not calling it report ?